test/scripts/download.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+if [ -d "../../dataset" ] && [ -f "../../dataset/dblpv13.json" ]
+then
+    exit 0
+else
+    if ! [ -d "./dblp.v13.7z" ]
+    then
+        mkdir "../../dataset"
+    fi
+
+    if ! [ -f "./dblp.v13.7z" ]
+    then
+        wget https://originalstatic.aminer.cn/misc/dblp.v13.7z
+    fi
+fi
+
+apt-get update
+apt-get install apt-get install p7zip-full
+7z x dblp.v13.7z -o../../dataset/
+rm dblp.v13.7z
```

Comment:
probably save this file?

Reply:
And check, if we have it in our directory, we need only unzip and sed. Seems logic

Reply:
But still, this archive is very big